aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -491,6 +492,22 @@ func resourceAwsKinesisFirehoseDeliveryStream() *schema.Resource {
 							Optional: true,
 						},
 
+						"s3_backup_mode": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Default:  "Disabled",
+							ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
This can be simplified with:
```go
ValidateFunc: validation.StringInSlice([]string{
  firehose.S3BackupModeDisabled,
  firehose.S3BackupModeEnabled,
}, false),
```